### ISSUE TITLE
Training from scratch

### ISSUE THREAD

**Issue by eslambakr** [experiencor/basic-yolo-keras#271]
Hi,
First of all thanks very much for this great work.

- I want to make an object detection system using mobile-net back-end but i want to make some changes in the model like i want to make the grid for example 14 * 14 instead of 7 * 7 
so i download the imagenet data and start the training for 20 epochs, i have GTX-1080 the training process for one epoch take 5 hours so i stopped it after 20 epochs, the training_acc= 75% and testing_acc= 54%

- Then i used this weights as pretrained weights for the detection task, 
- I am using COCO data set for training but make the labels for only 5-classes to make the training faster

learning rate=0.0001 /  Batch_size=64 /  NO_OBJECT_SCALE  = 0.5   /  OBJECT_SCALE     = 1.0
COORD_SCALE      = 5.0    /   CLASS_SCALE      = 1.0   
LABELS = ['person', 'bicycle', 'car', 'motorcycle', 'airplane']

1- warmup_epoch=1(i dont know should i use warmup epochs even when i start training coco from scratch or we should use it only when we have the coco as pretrained weights and we want to train a new class like kangroo in your example?)
Then i put the warmup to 0 and  start the training for 15 epochs but the result as following
Loss XY 	[0.307676643]
Loss WH 	[23.9863205]
Loss Conf 	[0.000186328791]
Loss Class 	[0.22972104]
Total Loss 	[24.5239048]
Current Recall 	[0]
Average Recall 	[1.12465987e-05]
2- Is this result normal?Current recall =0!
3- should i be more patient and train the detection for more epochs?(how many epochs is enough?)
4- the loss WH =23/5=4.6 i think it is very high, i think that's why the current recall is 0
    so in-which value of loss WH the current recall will start increasing?

Thanks in advance,
